guides/source/association_basics.md

```diff
@@ -1049,7 +1049,7 @@ There may be times when you wish to customize the query used by `belongs_to`. Su
 ```ruby
 class Book < ApplicationRecord
   belongs_to :author, -> { where active: true },
-                        dependent: :destroy
+                        touch: :books_updated_at
```

Comment:
I vote to remove this line, because its not directly relevant to the example of scoping (and neither is `dependent`).

Reply:
Agreed. Neither touch nor dependent has anything to do with scoping. It makes sense to restrict the scoping example to show what the scoping syntax looks like. Let me remove it

Reply:
Thank you. Please squash your commits and then I'll 🚢

Reply:
Done . Thanks